inst/@sym/solve.m

```diff
@@ -101,11 +120,10 @@
             '    else:'
             '        symbols.append(arg)'
             '#'
-            'if len(symbols) > 0:'
-            '    d = sp.solve(eqs, symbols, dict=True)'
-            'else:'
-            '    d = sp.solve(eqs, dict=True)'
+            'd = sp.solve(eqs, *symbols, dict=True)'
             '#'
+            'if not isinstance(d, (list, tuple)):'
+            '    return d,'
```

Comment:
Hi, when we works with ineq Sympy some times don't return lists with solutions and only send an expression with some conditions, so that case we need send that or we will get an error in the next steps of the script, sadly this is related with the non sense issue of here https://github.com/cbm755/octsympy/issues/581, this was on of the reasons of the assert in the  refactor.